server/plugins/client/bundle.js

```diff
@@ -14,20 +28,27 @@ var parallel = require('async').parallel
  * and avoiding unneeded bundling with browserify saves a significant time.
  */
 function bundleClient (hoodieClientPath, bundleTargetPath, config, callback) {
-  parallel([
+  var plugins = [
+    path.resolve(config.paths && config.paths.applicationRoot || '', 'hoodie/client')
+  ].filter(checkModule)
+  var getPluginsModifiedTimes = plugins.map(function (pluginPath) {
+    return getModifiedTime.bind(null, requireResolve(pluginPath))
+  })
```

Comment:
How about we add a section above `## Testing` and call it `## Extending Hoodie for your app` or something similar? 

It will also go into our docs, but that is another construction site :D